src/electron/future/main.ts

```
@@ -324,17 +326,23 @@ async function populateVectorStoreFromDocuments() {
 const apps: Record<string, BrowserWindow | null> = {};
 
 async function runStartup(withDashboard?: boolean) {
-	const apiKey = keyStore.get("openAiApiKey", "");
+	env.localModelPath = getCaptainDownloads("llm/embeddings");
+	env.allowRemoteModels = false;
+	env.allowLocalModels = true;
+
 	await VectorStore.init(
-		new OpenAIEmbeddings({
-			openAIApiKey: apiKey,
-			modelName: "text-embedding-3-large",
+		new CustomHuggingFaceTransformersEmbeddings({
+			modelName: "Xenova/all-MiniLM-L6-v2",
+			maxTokens: 128,
+			stripNewLines: true,
 		})
 	);
 
-	//
-	// await VectorStore.getInstance.deleteCollection(VECTOR_STORE_COLLECTION);
-	// await populateVectorStoreFromDocuments();
+	try {
+		await VectorStore.getInstance.deleteCollection(VECTOR_STORE_COLLECTION);
+	} catch {}
```

no empty catch. use `finally` instead